bucket/audioswitcher.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/xenolightning/AudioSwitcher_v1",
+    "description": "Audio Switcher (https://audioswit.ch/er) makes switching between sound devices trivial. No longer do you have to go into Control Panel or the Windows® Sound options, instead there is an easy to access icon, or even hotkeys.",
+    "version": "1.8.0.142",
+    "license": "MS-PL",
+    "url": "https://github.com/xenolightning/AudioSwitcher_v1/releases/download/1.8.0.142/AudioSwitcher.zip",
+    "hash": "e675b85a7c8465e02b58148f8d98b4adab78c3a7423160ca78d1237d8469ace4",
+    "bin": "AudioSwitcher.exe",
+    "shortcuts": [
+        [
+            "AudioSwitcher.exe",
+            "Audio Switcher"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/xenolightning/AudioSwitcher_v1/releases/download/$version/AudioSwitcher.zip"
+    },
+    "pre_install": "Stop-Process -Name AudioSwitcher -Verbose -ErrorAction Ignore",
+    "post_install": "Start-Process AudioSwitcher.exe -Verbose -NoNewWindow"
```

Comment:
Do not start it automatically. No all users would want to start applications automatically after installing.

```suggestion
```